tests/serde/compression/files.luau

```diff
@@ -37,6 +39,11 @@ for _, test in TESTS do
 	local source = fs.readFile(test.Source)
 	local target = fs.readFile(test.Target)
 
+	if isWindows then
+		-- Windows: CRLF -> LF
+		source = string.gsub(source, "\r\n", "\n", #source - 2);
```

Comment:
There might be a better way to do this using `.gitattributes` - but since we use many different formats here that might get a bit messy - so I think this is fine.

Reply:
Perhaps so, something like this `* text eol=lf`?

Reply:
I think that should work - could you double check what "text" includes in git terminology and make sure it covers our test files?

Reply:
I think the best way to do it would be `*.txt eol=lf` similar to the luau and lua files that already exists in the .gitattributes.

using `text` to my knowledge likely means to normalize commit files, I believe I'm not 100% sure.